playing_with_time.rb

```diff
@@ -0,0 +1,20 @@
+Time.now 
```

Comment:
I guess this is a work in progress file? :thinking:

Reply:
that was just a file I was using to work out how to get the time to save .. I should have deleted it